components/dashboard/src/settings/Billing.tsx

```diff
@@ -13,15 +13,16 @@ import { AttributionId } from "@gitpod/gitpod-protocol/lib/attribution";
 
 export default function Billing() {
     const { user } = useContext(UserContext);
-    const attributionId: AttributionId = { kind: "user", userId: user?.id || "" };
 
     return (
         <PageWithSettingsSubMenu title="Billing" subtitle="Usage-Based Billing.">
             <div>
                 <h3>Billing Account</h3>
```

Comment:
This could be quite confusing to users, especially now that we've removed the projects under personal accounts and moved the workspaces menu to the left, right?

If it takes more than a few minutes, let's open a follow-up issue for this! 🏓

Reply:
To visualize the issue here better, let me post two completely different areas in the dashboard using the same navigation state.

| Billing in User Settings | Projects in Team |
|-|-|
| <img width="1440" alt="Screenshot 2022-10-07 at 3 08 59 PM (2)" src="https://user-images.githubusercontent.com/120486/194550252-d944a9cf-daf0-4321-a328-8906ff355540.png"> | <img width="1440" alt="Screenshot 2022-10-07 at 3 09 02 PM (2)" src="https://user-images.githubusercontent.com/120486/194550292-73d40e4a-0e98-420a-b1e1-1e462663c610.png"> |

Reply:
I agree about the confusing part. What would be the correct fix?
- Show whichever team you had selected, but in an "inactive" (i.e. greyed out state)?
- Hide the team selector entirely on user settings pages?

Reply:
> Show whichever team you had selected, but in an "inactive" (i.e. greyed out) state?

Yes, which is exactly what we show on every other page under user settings, see example screenshots below.

Later on, we can discuss hiding, replacing, or improving navigation elements, see [relevant discussion](https://gitpod.slack.com/archives/C01KPEPNBD0/p1665147469413099?thread_ts=1665041540.568379&cid=C01KPEPNBD0) (internal).

| Notifications ✔️ | Variables ✔️ | SSH Keys ✔️ | Billing :x: |
|-|-|-|-|
| <img width="1440" alt="Screenshot 2022-10-07 at 3 54 52 PM (2)" src="https://user-images.githubusercontent.com/120486/194558716-e5cd0dc6-a106-4222-8de2-e786b4f642d2.png"> | <img width="1440" alt="Screenshot 2022-10-07 at 3 54 58 PM (2)" src="https://user-images.githubusercontent.com/120486/194558749-8c174348-9ad6-4679-ba78-b62678ec17cf.png"> | <img width="1440" alt="Screenshot 2022-10-07 at 3 54 59 PM (2)" src="https://user-images.githubusercontent.com/120486/194558763-3ee96dbc-14f5-4384-83c2-67fe7fd2fc90.png"> | <img width="1440" alt="Screenshot 2022-10-07 at 3 54 56 PM (2)" src="https://user-images.githubusercontent.com/120486/194558774-f00e2009-1a9b-4f4c-8097-51884a772337.png"> |